### ISSUE TITLE
Instrument ComboBox with a11y

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#333]
From phetsims/a11y-research#64, we would like to add a11y to ComboBox. I am going to take a first stab adding accessibility in a one-off branch because the focus highlight look, keyboard interaction, and output from screen reader are still being discussed. It was requested to implement an initial prototype to facilitate design decisions.

**Comment by jessegreenberg** [comment]
First pass at instrumentation has been added to the branch, we will want to clean this up before it is ready for master. But this should get us through discussion for a11y meeting on 1/26/18.

**Comment by jessegreenberg** [comment]
This is on hold until we hear more about how combobox should behave with  keyboard nav.

**Comment by pixelzoom** [comment]
Based on the title of this issue, it appears to be a duplicate of https://github.com/phetsims/sun/issues/314.   If it's somehow different, please change the title to reflect that.

**Comment by zepumph** [comment]
Indeed a duplicate of https://github.com/phetsims/sun/issues/314, closing.
